src/components/CreateWatch.svelte

```diff
@@ -0,0 +1,75 @@
+<script>
+import { getActiveFilterKeys } from '../data/filtering'
+import { getFiltersForWatch, getWatchableKeys } from '../data/watch'
+import { create } from '../data/watch'
+import { createdWatch } from '../data/analytics'
+import FilterTag from './FilterTag.svelte'
+import jquery from 'jquery' // $ is already a reserved token in Svelte (stores)
+import { onMount } from 'svelte'
+
+export let filter
+
+let name = ''
+let submitted = false
+
+function onSubmit() {
+  create(name, filter)
+  createdWatch()
+  submitted = true
+}
+
+onMount(async () => {
+    jquery('#thisModal').on('hidden.bs.modal', function (e) {
+        name = ''
+        submitted = false
+    })
+});
+
+$: canWatch = getWatchableKeys(getActiveFilterKeys(filter)).length
+$: watchFilters = getFiltersForWatch(filter)
+
+</script>
+{#if canWatch}
+  <button class={$$props.class} data-toggle="modal" data-target="#thisModal">Create Alert</button>
```

Comment:
```suggestion
  <button class={$$props.class} data-toggle="modal" data-target="#modal">Create Alert</button>
```

Reply:
since there's not another modal

Reply:
I think I read something that when using jquery methods on a modal, the id should be unique across the app. Even if that's not true, as a reader, I might be confused with "modal" as a Bootstrap identifier and "modal" as an app-specific identifier. (`data-toggle="modal"` and `data-target="modal"`) Do you mind if I leave it?